EpicLoot/src/Magic/MagicItemEffects/ModifyAttackHealth.cs

```diff
@@ -0,0 +1,25 @@
+﻿using HarmonyLib;
+
+namespace EpicLoot.MagicItemEffects
+{
+    [HarmonyPatch(typeof(Attack), nameof(Attack.GetAttackHealth))]
+    public class ModifyAttackHealth_Attack_GetAttackHealth_Patch
+    {
+        public static bool Prefix(Attack __instance, ref float __result)
```

Comment:
I think we just need to change m_attackHealthPercentage here. I recommend just changing that in the prefix and letting the original method run to potentially be more compatible with other mods.

Reply:
Finished in [commit](https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/5b885e9615288a1117bd17569c590ca2bd66dd9e)